types.d.ts

```diff
@@ -32,6 +32,8 @@ declare module "@logdna/logger" {
     sendUserAgent?: boolean;
     levels?: CustomLevel[];
     maxAttempts?: number;
+    verboseEvents?: boolean;
+    ignoreRetryableErrors?: boolean;
```

Comment:
Gah, thanks for catching this. We've also intended to put typescript validation/testing around this file at some point, but haven't gotten around to doing that.

Reply:
:+1: